### PR TITLE
Alt-DA: Add aliases to cli for Alt-DA

### PR DESCRIPTION
**Pull request by axelKingsley**
Makes `altda` or `ALTDA` the preferred prefix for Alt-DA flags, but keeps all existing flags as Aliases to preserve existing usage.

### CODE REVIEW

File: op-plasma/cli.go [hunk 1]
```diff
@@ -7,44 +7,55 @@ import (
...
+		envprefix + "_ALTDA_" + v,
+		envprefix + "_PLASMA_" + v}
 }
 
 func CLIFlags(envPrefix string, category string) []cli.Flag {
 	return []cli.Flag{
 		&cli.BoolFlag{
 			Name:     EnabledFlagName,
+			Aliases:  []string{EnabledFlagAlias},
 			Usage:    "Enable plasma mode\nPlasma Mode is a Beta feature of the MIT licensed OP Stack.  While it has received initial review from core contributors, it is still undergoing testing, and may have bugs or other issues.",
```
Comment (sebastianst) [inline op-plasma/cli.go]: "alt-da"?

File: op-plasma/cli.go [hunk 2]
```diff
@@ -7,44 +7,55 @@ import (
...
 			Usage:    "Verify input data matches the commitments from the DA storage service",
 			Value:    true,
-			EnvVars:  plasmaEnv(envPrefix, "VERIFY_ON_READ"),
+			EnvVars:  altDAEnvs(envPrefix, "VERIFY_ON_READ"),
 			Category: category,
 		},
 		&cli.BoolFlag{
 			Name:     DaServiceFlag,
+			Aliases:  []string{DaServiceFlagAlias},
 			Usage:    "Use DA service type where commitments are generated by plasma server",
```
Comment (sebastianst) [inline op-plasma/cli.go]: "alt-da"?

File: op-plasma/cli.go [hunk 3]
```diff
@@ -7,44 +7,55 @@ import (
...
+	return fmt.Sprintf("altda.%s", v), fmt.Sprintf("plasma.%s", v)
+}
+
+func altDAEnvs(envprefix, v string) []string {
+	return []string{
+		envprefix + "_ALTDA_" + v,
+		envprefix + "_PLASMA_" + v}
 }
 
 func CLIFlags(envPrefix string, category string) []cli.Flag {
```
Comment (sebastianst) [inline op-plasma/cli.go:29]: If you check where this fn is called, some string containing "plasma" might still be passed here. We should also rename the op-node category in that case.

File: op-plasma/cli.go [hunk 4]
```diff
@@ -7,44 +7,55 @@ import (
...
+	DaServerAddressFlagName, DaServerAddressFlagAlias = altDAFlags("da-server")
+	VerifyOnReadFlagName, VerifyOnReadFlagAlias       = altDAFlags("verify-on-read")
+	DaServiceFlag, DaServiceFlagAlias                 = altDAFlags("da-service")
 )
 
-func plasmaEnv(envprefix, v string) []string {
-	return []string{envprefix + "_PLASMA_" + v}
+// altDAFlags returns the flag names for altDA, with an Alias for plasma
+func altDAFlags(v string) (string, string) {
+	return fmt.Sprintf("altda.%s", v), fmt.Sprintf("plasma.%s", v)
```
Comment (sebastianst) [inline op-plasma/cli.go]: could also use simple concatenation
```suggestion
	return "altda." + v, "plasma." + v
```

**Review by sebastianst — COMMENTED** [review]
Looks good, when tests and comments are fixed. Better also check that we don't mention the old plasma flags in any READMEs or other docs in the monorepo.

File: op-plasma/cli.go (hunk 3 above)
Comment (axelKingsley) [reply op-plasma/cli.go:29]: `plasma.CLIFlags` is called only by OP-Node or OP-Batcher flag-creation, and in OP-Node, the `category` had been called Plasma, so good catch. Renaming to Alt-DA.

File: bedrock-devnet/devnet/__init__.py [hunk 5]
```diff
@@ -297,7 +297,7 @@ def devnet_deploy(paths):
         log.info('Bringing up `op-challenger`.')
         run_command(['docker', 'compose', 'up', '-d', 'op-challenger'], cwd=paths.ops_bedrock_dir, env=docker_env)
 
-    # Optionally bring up Plasma Mode components.
+    # Optionally bring up Alt-DA Mode components.
     if DEVNET_PLASMA:
```
Comment (sebastianst) [inline bedrock-devnet/devnet/__init__.py:301]: I guess env vars like this will be changed in a follow-up?

**Review by sebastianst — APPROVED** [review]

File: bedrock-devnet/devnet/__init__.py (hunk 5 above)
Comment (axelKingsley) [reply bedrock-devnet/devnet/__init__.py:301]: Yeah, we should do a separate pass later on to catch semi-visible things like this. The goal with this PR was to amend the CLI/Env flags for the binaries, as those are the most visible.

For `DEVNET_PLASMA`, there is also a `GENERIC_PLASMA` flag being used for different flag setting. It's something I'm not familiar enough with to make significant changes on without some more time reading and understanding.

**Review by ajsutton — APPROVED** [review]

**Pull request merged**